_component_design/table-pagination.md

```diff
@@ -0,0 +1,303 @@
+---
+title: Table Pagination
+parent: Tables
+layout: component
+category: Components
+usage: >
+  Table pagination refers to the controls that enable a user to traverse large data sets by viewing one page of table data at a time. Each page of data contains a set number of table rows (for example, 10). Users can navigate to Next and Previous pages.  
+preview-image: preview-images/tables-controls.svg
+status: stable
+resource: true
+last-modified: 2018-11-28
+helix-ui-css: false
+helix-ui-javascript: false
+---
+
+{% include toc.html %}
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Consider using pagination when a table spans multiple pages. You can also address large data sets by adding a **Load More** button that when clicked, scrolls the next group of table results up and into view.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+Consider the following best practices when you construct a pagination bar:
+
+- xxxxx.
```

Comment:
Make sure that you default load the appropriate number of records per page, and based on the contextual information needs of the user and the workflow.

Generally 10 records is a reasonable default number of records. You can also offer the user to change the number of records to display (using row controls, xref).

Ensure that you have the same number of records on each page until you run out of records. No 1, 5, 7 for 3 different pages.

See current content for a few words about number of page buttons.

Reply:
Done.